app/controllers/events_controller.rb

```diff
@@ -83,6 +88,80 @@ def schedule
     @schedule_type = @event.type.downcase!
   end
 
+  # GET /events/1/ranking
+  def ranking
```

Comment:
@bakoe @jonasnoki was sagt ihr zu dem jetzigen kontrollfluss? Mir erschien es am lesbarsten, den ranking_entry als parameter durchzureichen und dort die daten einzutragen...
Eine andere denkbare Möglichkeit wäre, recht viel der Logik dem Match zugeben. Dann hätte das Match Methoden wie `#get_ranking_entry_for_home_team` oder so... Was sagt ihr?

Reply:
As mentioned in my PR feedback, I approve the current control flow. I would prefer to not move the helper methods into the `Match` class since this would require that one more component knows about the RankingEntry Struct and knows how to handle it properly. 

Maybe a future user story will require to re-use the RankingEntry-specific methods of the EventsController in some other controller – to my mind, this would be a good time to factor out the methods into the `Match` class. But as long as this encapsulation is not needed and the RankingEntry Structs get only used within the `Event` scope, I would leave the methods were they are currently.